ember-server/shared/src/main/scala/org/http4s/ember/server/internal/WebSocketHelpers.scala

```diff
@@ -234,6 +236,49 @@ private[internal] object WebSocketHelpers {
       go(stream, Array.emptyByteArray).void.stream
     }
 
+  private def aggregateFragment[F[_]]: Pipe[F, WebSocketFrame, WebSocketFrame] =
```

Comment:
Do you think we should add some dedicated tests for this method? Actually I think it is not even specific to Ember, we can probably move it to http4s-core module and test it in-memory, instead of over a live connection.

Reply:
I just moved it to the core module.

> add some dedicated tests

Of cource, I'll add more tests!